x-pack/plugins/security/public/nav_control/nav_control_service.tsx

```diff
@@ -141,4 +149,40 @@ export class SecurityNavControlService {
   private sortUserMenuLinks(userMenuLinks: UserMenuLink[]) {
     return sortBy(userMenuLinks, 'order');
   }
+
+  private async telemetryOnAuthType(core: Pick<CoreStart, 'http'>) {
+    try {
+      const { BroadcastChannel } = await import('broadcast-channel');
+      const authenticatedUser = await this.authc.getCurrentUser();
+
+      if (!this.channel) {
+        const tenant = core.http.basePath.serverBasePath;
+        this.channel = new BroadcastChannel(`${tenant}/${USER_IN_SESSION}`, {
+          webWorkerSupport: false,
+        });
+      }
+
+      const elector = createLeaderElection(this.channel, {
+        fallbackInterval: 0, // optional configuration for how often will renegotiation for leader occur -> no need of renegotiation
+        responseTime: 1000, // optional configuration for how long will instances have to respond
+      });
+      elector.awaitLeadership().then(() => {
+        if (elector.isLeader) {
+          core.http.post('/internal/security/telemetry/auth_type', {
+            body: JSON.stringify({ auth_type: authenticatedUser.authentication_type }),
+          });
+        }
+      });
+
+      setTimeout(async () => {
+        if (!elector.isLeader) {
+          // Let's kill the non leader since we do not need to do telemetry if you are not a leader
+          // but we are keeping the leader alive to tell the other one that they won't be leader
+          await elector.die();
```

Comment:
I am wondering, If it will be a good idea to close the channel here too. I will do some test to make sure.